.gitignore

```diff
@@ -13,6 +13,7 @@ data
 
 # used in test
 tmp
+temp
```

Comment:
tmp or temp?

Reply:
I actually fixed this altogether by getting the test to write in /tmp, so I'll remove these 2